package.json

```diff
@@ -1,53 +1,107 @@
 {
   "name": "discover-dapps",
   "homepage": "https://dap.ps/",
-  "version": "0.1.0",
+  "version": "2.0.0",
   "private": true,
-  "dependencies": {
-    "@babel/runtime-corejs2": "^7.4.3",
+  "main": "index.js",
+  "scripts": {
+    "fullbuild:clean": "rm -rf ./full-build",
+    "fullbuild:init": "[ -d full-build ] || mkdir -p full-build",
+    "fullbuild:misc": "cp .npmrc full-build/",
+    "fullbuild:backend": "cp -r Backend/* full-build/",
+    "fullbuild:webapp": "mkdir full-build/frontend && cp -r WebApp/build/* full-build/frontend/",
+    "fullbuild:archive": "cd full-build/ && bestzip ../app.zip * && cd ..",
+    "create:fullbuild": "yarn build && yarn fullbuild:clean && yarn fullbuild:init && yarn fullbuild:misc && yarn fullbuild:backend && yarn fullbuild:webapp && yarn fullbuild:archive",
+    "build-internal": "yarn wsrun --stages --exclude-missing -c build",
+    "build": "dotenv yarn build-internal",
+    "build:local": "dotenv yarn build",
+    "build:webapp-internal": "yarn wsrun -p webapp -c build",
+    "build:webapp": "dotenv yarn build:webapp-internal",
+    "start:dev-internal": "yarn wsrun --parallel --exclude-missing start:dev",
+    "start:dev": "dotenv yarn start:dev-internal",
```

Comment:
This assumes that `dotenv` is available in `PATH`. Why not use `./node_modules/.bin/dotenv`?

Reply:
When running from within the npm script, its my understanding that it defaults to pulling from the local repository's node modules.

I dont have Embark for example installed globally